### ISSUE TITLE
Mobile support?

### ISSUE THREAD

**Issue by astartsky** [SystemSculpt/obsidian-systemsculpt-ai#10]
Greatly appreciate the amazing work with this plugin. 
Are there any plans to support it on mobile version of obsidian? Would be very handy to transcribe on the go.

**Comment by grayerbeard** [comment]
I agree or although I suspect it would need to be very different from the Desktop Version.  I would also like it if I could access a home server running Ollama from my mobile. I am not sure but I dont think any of the AI related PlugIns work on mobile at the moment. Even a reduced functionaly version would be invaluable.

**Comment by SystemSculpt** [comment]
mobile support for systemsculpt is very important to me, the only thing is i want to really build out all the features for the desktop version first, which will then allow me to convert them into a fully working mobile version afterwards.

though, these comments from you two really make me want to release some sort of lite version for mobile before i complete the desktop version, simply for transcription and possibly some sort of generation on the mobile device itself. apart from all of the recorder features, what else would be valuable to include in a mobile / lite version?

**Comment by grayerbeard** [comment]
I have never got around to using the recording transcription version on desktop but I would use it on a mobile version.
My most frequent way to use the plugin is to insert some text into a note and then add a sentence telling the AI what I want.  Also less often just typing in a prompt.  It would be great to have that on phone.
The other frequent use of obsidian on mobile is looking at my task list. (I use the Tasks plugin to list them in due order) Or adding a task and then using tasks plugin to set due date.
Of course on mobile there is no easy way to use local LLM (except when at home) but I am wondering if it might be possible to have ones own AI server at home accessable remotely.  As privacy concerns grow there might be demand for that - a longer term thing perhaps.

Great to hear your positive response thanks a lot.

DT

**Comment by astartsky** [comment]
Hi grayerbeard! 
I think most interesting feature for mobile would be a chat with a note. I always have some thoughts on the go and it would be very handy to be able to talk with it a bit. Second priority for me would be a transcriber as it's also very handy on the go.

> Of course on mobile there is no easy way to use local LLM (except when at home) but I am wondering if it might be possible to have ones own AI server at home accessable remotely. 

It can be done with Tailscale. With it, you can setup a secure virtual network between your devices and use their dns names to talk to each other no mater what their actual network is. I use it all the time to access Ollama instance on my home computer.

**Comment by grayerbeard** [comment]
Thanks ref Tailscale, I remember there was something beginning with T.  Others are TeamViewer, Dataplicity, RealVNC, RDP, Chrome DeskTop, AnyDesk of which I believe only some are suitable for our purpose connecting a phone to a remote LLM server.
I hope what will be possible is a very private personal assistant I can take anywhere that knows about me because it has access to my "Second Brain".  A brain which, I dream, as I get older and forget more and more will help me remember.
This is the sort of thing that counters the anxiety of many that a super server cluster costing billions consuming gigawatts will become the super big brother to us all.

DT
Grayerbeard

**Comment by olafgeibig** [comment]
A mobile version is absolutely essential. In the 21st century many people don't own a computer anymore but almost everybody own a mobile device. This is very true for Gen Z and even more true for poorer countries. My 17 yo son for instance. Myself, I primarily use Obsidian on my iPad. A second brain is much more useful if you can use it all the time and everywhere you go. Please revisit your decision. Mobile first is normal in the industry.

**Comment by SystemSculpt** [comment]
Thank you for the comments and thoughts on this matter, I see there is heavy interest in a mobile version and I will speed up the timeline regarding its production schedule. I'll keep things updated in here and the issue opened for the time being until I have a rough version working and ready for testing.

**Comment by zdaar** [comment]
Hey, big Systemsculpt user here, it is core to my obsidian workflow nowadays. I would absolutely test and contribute to a mobile version. Voice transcript here being the main use case.

**Comment by LePenseur13** [comment]
Mobile would be much appreciated

**Comment by grayerbeard** [comment]
Yes but wouldn't iit need to be a slightly simplified version (or automatically disabled some things when on mobile). After all a phone would struggle to cope with multiple chat windows?

I often use one of the mobile Apps such as Perplexity, Gemeni, Copilot etc and then copy paste the result into Obsidian on my phone.  If I could do that on my phone with SystemSculpt it would be much better especially as files are more effectively added for context.

I am looking at accessing my home AI server remotely.

**Comment by SystemSculpt** [comment]
[Automated] Closing in preparation of version 1.0 beta release. If this issue persists after updating, please reopen with details.
